src/fedora/41/helix/Dockerfile

```
@@ -0,0 +1,74 @@
+FROM library/fedora:41 AS venv
+
+RUN dnf upgrade --refresh -y && \
+    dnf install --setopt tsflags=nodocs -y \
+        dnf-plugins-core && \
+    dnf install --setopt=install_weak_deps=False --setopt tsflags=nodocs -y \
+        openssl \
+        python3-devel \
+        python3-pip \
+        gcc \
+        libatomic \
+        redhat-rpm-config && \
+    dnf clean all
```

Please follow this convention instead which is what we're trying to consolidate on for all the Dockerfiles. This should get applied consistently for the rest of this Dockerfile as well.

```suggestion
RUN dnf upgrade --refresh -y \
    && dnf install --setopt tsflags=nodocs -y \
        dnf-plugins-core \
    && dnf install --setopt=install_weak_deps=False --setopt tsflags=nodocs -y \
        openssl \
        python3-devel \
        python3-pip \
        gcc \
        libatomic \
        redhat-rpm-config \
    && dnf clean all
```